### PR TITLE
[PL] Add some expressions to timers

### PR DESCRIPTION
**Pull request by spitfire**

### CODE REVIEW

**Review by Uriziel01 — CHANGES_REQUESTED** [review]

**Comment by Uriziel01** [comment]
Way to go GitHub, thanks for informing me in the notification this is once again a bloody draft PR...

File: sentences/pl/_common.yaml [hunk 1]
```diff
@@ -630,14 +630,14 @@ expansion_rules:
   what_is_the_class_of_name: "(<what_is> <class> [z [urzadzenia]|w [urządzeniu]|urządzenia|czujnika|wskaz(uje|any|ywan[a|e|y|ego] [przez])|zmierzone przez [czujnik]] <name> [<area>]|<what_is> <name> <class> [<area>]|<what_is> <area> <name> <class>)"
 
   # Timers
-  timer: "(minutnik[a|i|u|ów]|zegar[a|ze|y|ów]|tajmer[a|ze|y|ów]|licznik[a|u|i|ów]|czasomierz[a|u|e|y]|stoper[a|ze|y|ów])"
-  timer_plural: "(minutnik[ów|ach]|zegar[ów|ach]|tajmer[ów|ach]|licznik[ów|ach]|czasomierz[y|ach]|stoper[ów|ach])"
+  timer: "(minutnik[a|i|u|ów]|zegar[a|ze|y|ów]|tajmer[a|ze|y|ów]|timer[a|ze|y|ów]|licznik[a|u|i|ów]|czasomierz[a|u|e|y]|stoper[a|ze|y|ów])"
```
[the remark this answers is not in the document]
Comment (spitfire) [reply sentences/pl/_common.yaml:633]: I'll try, thanks for pointing it out:)

[the remark this answers is not in the document]
Comment (spitfire) [reply sentences/pl/_common.yaml:633]: Is it okay now?

[the remark this answers is not in the document]
Comment (Uriziel01) [reply sentences/pl/_common.yaml:633]: I've just approved a tests run, let's see 😉

[the remark this answers is not in the document]
Comment (spitfire) [reply sentences/pl/_common.yaml:633]: I can see where I got it wrong, give me a moment..

[the remark this answers is not in the document]
Comment (spitfire) [reply sentences/pl/_common.yaml:633]: Ok, I don't know what went wrong the last time. I didn't add `zatrzymaj` yet it failed with:
`FAILED tests/test_language_sentences.py::test_homeassistant_HassPauseTimer[pl] - AssertionError: For 'zatrzymaj minutnik' expected intent HassPauseTimer, got HassCancelTimer`

**Pull request closed without merging**